src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -517,9 +516,9 @@ internal void EnsureNullableAttributeExists(DiagnosticBag? diagnostics, Location
             EnsureEmbeddableAttributeExists(EmbeddableAttributes.NullableAttribute, diagnostics, location, modifyCompilation);
         }
 
-        internal void EnsureNullableContextAttributeExists(DiagnosticBag? diagnostics, Location location, bool modifyCompilation)
+        internal void EnsureNullableContextAttributeExists(DiagnosticBag? diagnostics, Location location)
```

Comment:
>internal void EnsureNullableContextAttributeExists(DiagnosticBag? diagnostics, Location location) [](start = 8, length = 97)

This refactoring doesn't look related to the primary goal of this PR and the feature in general. Please exclude it from the PR.

Reply:
I suggest to revert the entire file.

---
In reply to: [410997710](https://github.com/dotnet/roslyn/pull/43275#discussion_r410997710) [](ancestors = 410997710)